src/components/SearchResults.js

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from "react";
+import moment from "moment";
+
+export default class SearchResults extends Component {
+  constructor() {
+    super();
+    this.state = {};
+  }
+
+  rowSelected = event => {
```

Comment:
I think this will simply select and deselect ALL rows any time ANY row is clicked.  Not what was specified in the HW task.